documentation/src/main/asciidoc/topics/ref_performance_tuning.adoc

```diff
@@ -79,6 +79,88 @@ The recommended values are:
 * send window size - 640 KB
 * receive window size - 25 MB
 
+== Thread pool settings
+
+A thread pool manages threads by assigning them concurrent tasks.
+Additionally, a thread pool can improve system performance by using any idle thread instead of creating threads.
+
+Thread pools limit the creation and deletion of threads to active worker threads, so that the amount of threads does not typically exceed the amount of tasks.
+This helps reduce the memory footprint of your {brandname} server.
+
+Your {brandname} configuration applies default settings to thread pools, but you can change these settings to meet your needs.
+
+The following table outlines common thread pool properties and their default settings:
+
+[cols="1,1,3"]
+|===
+| Property | Default setting | Description
+
+|`ASYNC_NOTIFICATION_EXECUTOR` |`1`| Used by asynchronous listeners that your {brandname} server registers in embedded mode. 
```

Comment:
I would say just use the name of the xml schema on the cache container configuration for the property. Technically this table only highlights the maximum thread size, there are other properties of the threads including priority, core threads, queue size. The defaults for these values are at https://github.com/infinispan/infinispan/blob/main/core/src/main/java/org/infinispan/factories/KnownComponentNames.java#L40-L54

Reply:
Thanks, @wburns . I'm getting confused here. `ASYNC_NOTIFICATION_EXECUTOR` is a thread pool or a thread pool property? In turn, `ASYNC_NOTIFICATION_EXECUTOR` contains further properties that configure thread behavior?

I updated the table. I placed `DEFAULT_THREAD_PRIORITY` in the intro line for the table, as each property shared the same thread property and its default value.

Reply:
Hi @wburns . I rearranged the information. I also mentioned the cache container configuration that exists inside the Infinispan Server's configuration. WDYT?